### PR TITLE
Issue/load url and link

### PR DESCRIPTION
**Pull request by tomfelix**

### CODE REVIEW

File: src/customerCalls/CallForm.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { CallFormDataList, CallFormInput, CallFormOption, CallFormWrapper } from
 interface State {
     query: string;
     networkType: string;
+    selectedOption: any;
```
Comment (przemyslawjanpietrzak) [inline src/customerCalls/CallForm.tsx]: You can do it better ;)

File: package.json [hunk 2]
```diff
@@ -14,11 +14,14 @@
...
     "@fortawesome/free-solid-svg-icons": "^5.10.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
+    "@types/react-select": "^3.0.2",
     "axios": "0.19.0",
+    "datalist-polyfill": "^1.23.3",
     "prop-types": "15.6.2",
     "react": "16.8",
     "react-dom": "16.8",
     "react-scripts": "3.0.1",
+    "react-select": "^3.0.4",
```
Comment (przemyslawjanpietrzak) [inline package.json]: where did you use it?

File: src/customerCalls/CallForm.tsx (hunk 1 above)
Comment (tomfelix) [reply src/customerCalls/CallForm.tsx]: should be removed, fixed

File: package.json (hunk 2 above)
Comment (tomfelix) [reply package.json]: should be removed, fixed

File: package.json [hunk 3]
```diff
@@ -14,7 +14,9 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.21",
     "@fortawesome/free-solid-svg-icons": "^5.10.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
+    "@types/react-select": "^3.0.2",
```
Comment (przemyslawjanpietrzak) [inline package.json]: again

**Pull request merged**